app/policies/cinema_hall_policy.rb

```diff
@@ -4,6 +4,7 @@ class CinemaHallPolicy < ApplicationPolicy
   attr_reader :user, :cinema_hall
 
   def initialize(user, cinema_hall)
+    super
```

Comment:
Rubocop suggested that change throwing: _Lint/MissingSuper: Call super to initialize state of the parent class._ I am not sure if it is really needed in this case. 🤔

Reply:
Corrected, thanks :) I must have misunderstood the following example in the documentation:

_class PostPolicy
  attr_reader :user, :post

  def initialize(user, post)
    @user = user
    @post = post
  end

  def update?
    user.admin? || !post.published?
  end
end_
[](https://github.com/varvet/pundit)

Reply:
In the mentioned example they don't use base class (like `ApplicationPolicy` in your example) so they have to define `initialize` method.

Reply:
@atytko consider if you really need `initialize` method if everything is already defined in [ApplicationPolicy](https://github.com/atytko/monte-cinema/blob/main/app/policies/application_policy.rb#L6)

This comment is applicable to all policies.

Reply:
You are right, I don't need it at all. :) I have removed the `initialize` methods inside my policies.